com.unity.render-pipelines.high-definition/Documentation~/Decal-Shader.md

```diff
@@ -2,7 +2,7 @@
 
 The High Definition Render Pipeline (HDRP) includes the [Decal Projector](Decal-Projector.html) component, which you can use to project specific Materials into your Scene to create realistic-looking decals. These Materials must use the **HDRP/Decal** Shader. Use this Shader to create decal Materials that you can place, or project, into your Scene.
 
-![](Images/HDRPFeatures-DecalShader.png)
+A Decal shader can be use on a [Decal Projector](Decal-Projector.html) or a Decal Mesh.
```

Comment:
What about the nice decal picture?

You can use a Decal Shader in a [Decal Projector](Decal-Projector.md) or on a Decal Mesh.

Reply:
arf, good catch for the picture :)